src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -199,4 +199,17 @@ public Metadata()
                     storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.ProvideDateAndTimeCompletions"));
         }
     }
+
+    internal static class CompletionOptionsExtensions
+    {
+        public static bool ShouldShowItemsFromUnimportNamspaces(this CompletionOptions options)
```

Comment:
>ShouldShowItemsFromUnimportNamspaces [](http://example.com/codeflow?start=27&length=36)

The naming is confusing. I don't see when I should use `ShouldShowItemsFromUnimportNamspaces()` vs `ShowItemsFromUnimportedNamespaces`.

Reply:
I have added some comment. And hopefully we can remove the experiment soon